Fix doc for idle (#19802)
nrfconnect/sdk-nrf: tests/benchmarks/multicore/idle/README.rst

     west build -p -b nrf54h20dk/nrf54h20/cpuapp -T benchmarks.multicore.idle.nrf54h20dk_cpuapp_cpurad .

>   To achive the lowest power consumption, please use configuration ``benchmarks.multicore.idle.nrf54h20dk_cpuapp_cpurad.s2ram``.
>   Please note that this configuration will not provide console output. Build command:

Reviewer: ```suggestion
  To achieve the lowest power consumption, use the ``benchmarks.multicore.idle.nrf54h20dk_cpuapp_cpurad.s2ram`` configuration.
  This configuration does not provide console output.
  Use the following build command:
```

Author: Thanks, changed as requested.
